packages/aztec.js/test/proof/bilateralSwap/proof.spec.js

```diff
@@ -1,32 +1,21 @@
 /* global, beforeEach, it:true */
-const BN = require('bn.js');
 const chai = require('chai');
-const crypto = require('crypto');
-const { padLeft, randomHex, sha3 } = require('web3-utils');
+const { randomHex } = require('web3-utils');
 
-const bn128 = require('../../../src/bn128');
 const bilateralProof = require('../../../src/proof/bilateralSwap');
-const helpers = require('../../../src/proof/bilateralSwap/helpers');
 const Keccak = require('../../../src/keccak');
-const { K_MAX } = require('../../../src/params');
-
+const proofUtils = require('../../../src/proof/proofUtils');
 
 const { expect } = chai;
 
 
-function generateNoteValue() {
-    return new BN(crypto.randomBytes(32), 16).umod(new BN(K_MAX)).toNumber();
-}
-
-
 describe('AZTEC bilateral swap proof construction tests', () => {
     let testNotes;
     let sender;
 
     beforeEach(() => {
-        testNotes = helpers.makeTestNotes([10, 20], [10, 20]);
+        testNotes = proofUtils.makeTestNotes([10, 20], [10, 20]);
```

Comment:
Same comment as on previous code reviews, better to have all state for tests be contained in the test case, even if slightly wordier.

Reply:
Good point, this is a legacy pattern - will update/remove this pattern as I come across it in the codebase